.circleci/config.yml

```diff
@@ -614,32 +616,49 @@ jobs:
           steps:
             - run:
                 name: Cypress
+                # Need to run from frontend directory, because cypress-circleci-reporter
+                # reports paths relative to there
                 command: |
                   cd frontend
-                  yarn run cy:serve-and-run-all-flaky
+                  TEST=$(circleci tests glob ../e2e/specs/*.spec.js | circleci tests split --split-by=timings)
```

Comment:
I assume this is the key magic that allows this whole thing to work - assuming that's correct, can you add a brief comment explaining what's happening here? ("`circleci tests split` splits our long-running Cypress e2e tests into N different parallel jobs, so that they finish much faster than if we run them in sequence" - or whatever.)